app.rb

```diff
@@ -0,0 +1,95 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './classroom'
+require_relative './rental'
+
+class App
+  attr_accessor :books, :students, :teachers, :rentals
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def display_all_books
+    @books.each_with_index do |book, index|
+      puts "#{index}) Title: '#{book.title}', Author: '#{book.author}'"
+    end
+  end
+
+  def display_all_people
+    @people.each_with_index do |people, index|
+      puts "#{index}) Name: #{people.name}, ID: #{people.id}, Age: #{people.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student(1) or a teacher(2)? [Input the number]: '
+    person_option = gets.chomp.to_i
```

Comment:
- [ ] please, like the previous reviewer noted, kindly add a case where a user enters an option that doesn't exist(!1/2). it will help not to break the app as well as make it user-friendly. 😄